lib/msf/base/sessions/meterpreter.rb

```diff
@@ -326,8 +326,10 @@ def load_session_info()
     begin
       ::Timeout.timeout(60) do
         # Gather username/system information
-        username  = self.sys.config.getuid
-        sysinfo   = self.sys.config.sysinfo
+        username = self.sys.config.getuid
+        sysinfo  = self.sys.config.sysinfo
+
+        self.payload_uuid = self.core.uuid
```

Comment:
Blindly overwriting this doesn't seem like a good idea (the uuid comes from the initial connection/staging process)